flypg/launcher.go

```diff
@@ -282,6 +282,11 @@ func (l *Launcher) getPostgresConfig(config *CreateClusterInput) *api.MachineCon
 		},
 	}
 
+	// medatata
```

Comment:
How do you propose we do it?

Reply:
Yeah, probably fine for now.  The two cases where it could lead to issues are:

1. If someone moves to a fork or transitions their image to something other than `flyio/postgres`.
This is fine, as the call to `client.GetLatestImageDetails(ctx, image)` will fail. 

2. If someone transitions away from a fork to our managed image.
This case is less ideal, as even though they are running our managed image the metadata will prevent the operation from going through.